doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+.. _rt_industry_setup:
+
+How to launch CL-RT VM with ACRN Industry scenario
```

Comment:
how about changing CL-RT VM to RTVM?

Reply:
Sure, I've replaced all the ``CL-RT`` to the ``RTVM``.